download_and_create_reference_datasets/v03/write_combined_reference_ht.py

```diff
@@ -67,10 +70,6 @@ def run(environment: str, genome_version: str, dataset: str):
         choices=['37', '38'],
         default='38',
     )
-    parser.add_argument(
-        '--dataset',
-        choices=DATASETS,
-        required=True,
-    )
+    parser.add_argument('--dataset', choices=DATASETS, default=None)
```

Comment:
Same thing with help.
```suggestion
    parser.add_argument('--dataset', choices=DATASETS, default=None, help="When passed, update the single dataset, otherwise update all datasets.")
```
Feature showed up here but feel free to rewrite that.